CHANGELOG.md

```diff
@@ -8,32 +8,78 @@ core section.***
 - ***When possible please provide the number of the pull request(s) containing the 
 changes in the following format: PR #1234***
 
-### LORIS 23.0 (Released: ??)
+## LORIS 23.0 (Release Date: ??)
 
+### Core
+#### Features
+- Add configuration option to specify the format in which dates are displayed. (PR #5004)
+- Configurable upload directory for data_release and document_repository modules (PR #5815)
+- New radio components for forms. (PR #5846)
 
-#### Core
+#### Updates and Improvements
+- PHP minimum requirements raised to PHP 7.3 (PR #5723)
```

Comment:
```suggestion
- PHP minimum requirements raised to PHP 7.3. (PR #5723)
```
since some line consist of multiple phrases, could you add ending dots to every lines?